features/frameworks/greentea-client/source/test_env.cpp

```diff
@@ -72,6 +73,7 @@ void GREENTEA_SETUP(const int timeout, const char *host_test_name) {
 	char _value[48] = {0};
 	while (1) {
         greentea_parse_kv(_key, _value, sizeof(_key), sizeof(_value));
+        greentea_write_string("mbedmbedmbedmbedmbedmbedmbedmbed\r\n");
```

Comment:
Is there a reason we are writing this back? I know we write a similar string from htrun, though I admit I'm not sure why :)